src/cordova/android/container/README.md

```diff
@@ -7,27 +7,28 @@ This approach is based on [microcontainers](https://www.iron.io/microcontainers-
 First, install Docker:
 https://docs.docker.com/engine/installation/
 
-Then, prefix your commands with: `./tools/build/build.sh`
+Then, prefix your commands with: `npm run action cordova/android/container/start`
+
+So, to install dependencies and build the Android App:
 
-So, to install dependencies and build:
 ```bash
-./tools/build/build.sh npm ci
-./tools/build/build.sh npm run build
+npm run action cordova/android/container/start npm ci
```

Comment:
I find the name "container" here counter intuitive. It feels like the android client runs on a container. Could you name it `android/build_container` instead?